docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,194 @@
+
+Now we can query for the tags on the given application.
+
+    newrelic entities describe-tags -g MjUwODI1OXxBUE18QVBQTElDQVRJT058MjA0MjYxMzY4
+
+Which gives the following output.
+
+    [
+      ...
+      {
+        "Key": "compliant",
+        "Values": [
+          "true"
+        ]
+      },
+      {
+        "Key": "env",
+        "Values": [
+          "test"
+        ]
+      },
+      ...
+      {
+        "Key": "zleslie",
+        "Values": [
+          "istesting"
+        ]
+      }
+    ]
+
+#### Creating a deployment marker
+
+To create a deployment, we use the same ApplicationID from our earlier search,
+and pass that to the `apm create-deployment` command.
+
+    newrelic apm create-deployment -a 204261368 -r $(git rev-parse HEAD)
+
+With this we get a response back that looks like the following.
+
+    {
+      "id": 37075986,
+      "links": {
+        "application": 204261368
+      },
+      "revision": "40af0d07d86363d289e772f4326d23d1717fb765",
+      "timestamp": "2020-03-04T15:11:44-08:00",
+      "user": "Developer Toolkit Test Account"
+    }
+
+You can see here that the revision that was recorded was the SHA from my git
+project.  The revision can be any string, but the intent is to provide some
+point in time for the application in question.
+
+This sort of thing could be built into a CI/CD system to help indicate changes
+in application behavior after deployments.
+
+#### Tagging all applications for an account
+
+Another use case, perhaps, is that you need to tag all applications in your
+environment.  This could be daunting to do through the web UI, but at the CLI
+this is pretty quick.
+
+Here we'll couple the `newrelic` CLI with the `jq`  command to tag each
+application returned in our query with a new tag.
+
+    newrelic apm application get --accountId $ACCOUNT_ID | jq -r '.[].GUID' | xargs -I {} newrelic entities create-tags -g {} -t zleslie:istesting
+
+In the above, we first run `apm application get` to return the applications for
+our given account.  This could be filtered further to reduce the set of
+results, but for illustration purposes this works well enough.
+
+Next, we pipe the JSON output containing all the results to `jq`.  The
+less-than-beautiful syntax there is picking out only the `GUID` field and
+printing each results GUID on its own line.
+
+Then we pass that to `xargs` to call the `newrelic` CLI to use the `entities
+create-tags` the received GUID strings so that each application gets tagged.
+
+[Jq][jq] can be a powerful way to manipulate JSON data at the command line.  If
+you're not familiar with it, its worth looking at to tie these sorts of
+commands together.
+
+
+### Want more?
+
+If you'd like to engage with other community members, visit our
+[discuss][discuss] page.  We welcome feature requests or bug reports on GitHub,
+or for the ambitious, you can send us pull requests also.
```

Comment:
The "for the ambitious" part kind of makes it feel like this could be a daunting task. Since we want to seem open and user-friendly, we might want to leave the ambitious part out 😆